### PR TITLE
Fix delete old lightmaps issue.

### PR DESCRIPTION
**Pull request by H1Gdev**
In Selected bake, can delete old lightmaps.

### CODE REVIEW

File: src/scene/lightmapper.js [hunk 1]
```diff
@@ -203,17 +203,17 @@ pc.extend(pc, function () {
 
                 // delete old lightmaps, if present
                 var k;
-                for(i=0; i<sceneLightmaps.length; i++) {
-                    for(i=j; j<nodes.length; j++) {
```
Comment (guycalledfrank) [inline src/scene/lightmapper.js]: Did I write this?... how did it even work before? :D

**Comment by daredevildave** [comment]
Hi @H1Gdev, could you complete the Contributor License Agreement: https://docs.google.com/a/playcanvas.com/forms/d/e/1FAIpQLSfG9XHIgdapCBkvwxbXow592Y5NFPEKD5TM-TyWOmbaCbLeHA/viewform

Then we can accept the PR.

Thanks,

File: src/scene/lightmapper.js (hunk 1 above)
Comment (H1Gdev) [inline src/scene/lightmapper.js]: j is undefined, so 'j<nodes.length' is always false.

**Comment by Maksims** [comment]
Thank you for contributing!

**Pull request merged**